source/lattice-construction.md

```diff
@@ -0,0 +1,158 @@
+(c:lattice-construct)=
+# Lattice Construction
+
+%---------------------------------------------------------------------------------------------------
+(s:lattice.construct)=
+## Constructing a Lattice
+
+A `Lattice` is essentially a set of branches. 
+Each branch is instantiated from a `BeamLine` which is called the a `root BeamLine`.
```

Comment:
Replace "which" with "that" and remove "a"?

Reply:
Done